### ISSUE TITLE
Docker does not start with Read Only Mounted Share

### ISSUE THREAD

**Issue by ghost** [filebrowser/filebrowser#1734]
**Description**
I have FileBrowser setup as a Docker Container, up until the most recent update (2.20) I had the Data mounted under `/srv` as Read Only, however following the update to 2.20, the container no longer starts.

The below can be seen in the Docker Logs;

```
[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 20-config: executing...
chown: changing ownership of '/srv': Read-only file system
[cont-init.d] 20-config: exited 1.
[cont-init.d] 90-custom-folders: executing...
[cont-init.d] 90-custom-folders: exited 0.
[cont-init.d] 99-custom-files: executing...
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-files: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
ifelse: fatal: unable to exec -d: No such file or directory
[cmd] -d exited 127
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

**Expected behaviour**
The Docker Container should not need to change permissions on `/srv` and the container should still start if the file system is read only (its there just to Browse Files and Download).

**What is happening instead?**
The Docker Container is Stopping/Exiting. See Log above.

**Additional context**
Was working up until most recent update.

**How to reproduce?**
Deploy as v2.19 via Docker, Mount your data to `/srv` as Read Only, verify it works. Update to v2.20 and look at Docker Logs.

**Files**
None

**Comment by tvaios** [comment]
I have similar problem after updating to the new image

Docker run:
`docker run -d \
    --name=filebrowser \
    --hostname=filebrowser \
    -p 8081:80 \
    -v /home/pi/:/srv \
    -v /home/pi/docker/filebrowser/config/filebrowser.db:/database.db \
    -v /home/pi/docker/filebrowser/config/filebrowser.json:/.filebrowser.json \
    --user $(id -u):$(id -g) \
    --net proxy \
    --restart="unless-stopped" \
    filebrowser/filebrowser:latest`

filebrowser.json
`{
  "port": 80,
  "baseURL": "",
  "address": "",
  "log": "stdout",
  "database": "/database.db",
  "root": "/srv"
}`

log
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
s6-chown: fatal: unable to chown /var/run/s6/etc/cont-init.d/20-config: Operation not permitted
s6-chown: fatal: unable to chown /var/run/s6/etc/cont-init.d/01-envfile: Operation not permitted
s6-chown: fatal: unable to chown /var/run/s6/etc/cont-init.d/99-custom-files: Operation not permitted
s6-chown: fatal: unable to chown /var/run/s6/etc/cont-init.d/10-adduser: Operation not permitted
s6-chown: fatal: unable to chown /var/run/s6/etc/cont-init.d/90-custom-folders: Operation not permitted
s6-chmod: fatal: unable to change mode of /var/run/s6/etc/cont-init.d/20-config: Operation not permitted
s6-chmod: fatal: unable to change mode of /var/run/s6/etc/cont-init.d/01-envfile: Operation not permitted
s6-chmod: fatal: unable to change mode of /var/run/s6/etc/cont-init.d/10-adduser: Operation not permitted
s6-chmod: fatal: unable to change mode of /var/run/s6/etc/cont-init.d/99-custom-files: Operation not permitted
s6-chmod: fatal: unable to change mode of /var/run/s6/etc/cont-init.d/90-custom-folders: Operation not permitted
s6-chown: fatal: unable to chown /var/run/s6/etc/services.d/filebrowser/run: Operation not permitted
s6-chmod: fatal: unable to change mode of /var/run/s6/etc/services.d/filebrowser/run: Operation not permitted
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 01-envfile: executing... 
foreground: warning: unable to spawn /var/run/s6/etc/cont-init.d/01-envfile: Permission denied
[cont-init.d] 01-envfile: exited 127.
[cont-init.d] 10-adduser: executing... 
foreground: warning: unable to spawn /var/run/s6/etc/cont-init.d/10-adduser: Permission denied
[cont-init.d] 10-adduser: exited 127.
[cont-init.d] 20-config: executing... 
foreground: warning: unable to spawn /var/run/s6/etc/cont-init.d/20-config: Permission denied
[cont-init.d] 20-config: exited 127.
[cont-init.d] 90-custom-folders: executing... 
foreground: warning: unable to spawn /var/run/s6/etc/cont-init.d/90-custom-folders: Permission denied
[cont-init.d] 90-custom-folders: exited 127.
[cont-init.d] 99-custom-files: executing... 
foreground: warning: unable to spawn /var/run/s6/etc/cont-init.d/99-custom-files: Permission denied
[cont-init.d] 99-custom-files: exited 127.
[cont-init.d] done.
[services.d] starting services
s6-supervise (child): fatal: unable to exec run: Permission denied
s6-supervise filebrowser: warning: unable to spawn ./run - waiting 10 seconds
[services.d] done.
```

**Comment by the-lazy-fox** [comment]
@tvaios  I have the same as you, but not related to read only mount: https://github.com/filebrowser/filebrowser/issues/1736

**Comment by ghost** [comment]
In v2.20.1 (where it reverts back to an Alpine image), I can confirm that the container now starts again as expected with a Ready Only Mount.

**Comment by o1egl** [comment]
Fixed in v2.20.1
